Basic/Binding/Basic/Http/CS/Client/Connected Services/CoreWcf.Samples.Http/Reference.cs

```diff
@@ -0,0 +1,183 @@
+        
+        public System.Threading.Tasks.Task<double> AddAsync(double n1, double n2)
+        {
+            return base.Channel.AddAsync(n1, n2);
+        }
+        
+        public double Subtract(double n1, double n2)
+        {
+            return base.Channel.Subtract(n1, n2);
+        }
+        
+        public System.Threading.Tasks.Task<double> SubtractAsync(double n1, double n2)
+        {
+            return base.Channel.SubtractAsync(n1, n2);
+        }
+        
+        public double Multiply(double n1, double n2)
+        {
+            return base.Channel.Multiply(n1, n2);
+        }
+        
+        public System.Threading.Tasks.Task<double> MultiplyAsync(double n1, double n2)
+        {
+            return base.Channel.MultiplyAsync(n1, n2);
+        }
+        
+        public double Divide(double n1, double n2)
+        {
+            return base.Channel.Divide(n1, n2);
+        }
+        
+        public System.Threading.Tasks.Task<double> DivideAsync(double n1, double n2)
+        {
+            return base.Channel.DivideAsync(n1, n2);
+        }
+        
+        public virtual System.Threading.Tasks.Task OpenAsync()
+        {
+            return System.Threading.Tasks.Task.Factory.FromAsync(((System.ServiceModel.ICommunicationObject)(this)).BeginOpen(null, null), new System.Action<System.IAsyncResult>(((System.ServiceModel.ICommunicationObject)(this)).EndOpen));
+        }
+        
+        public virtual System.Threading.Tasks.Task CloseAsync()
+        {
+            return System.Threading.Tasks.Task.Factory.FromAsync(((System.ServiceModel.ICommunicationObject)(this)).BeginClose(null, null), new System.Action<System.IAsyncResult>(((System.ServiceModel.ICommunicationObject)(this)).EndClose));
+        }
+        
+        private static System.ServiceModel.Channels.Binding GetBindingForEndpoint(EndpointConfiguration endpointConfiguration)
+        {
+            if ((endpointConfiguration == EndpointConfiguration.BasicHttpBinding_ICalculatorService))
+            {
+                System.ServiceModel.BasicHttpBinding result = new System.ServiceModel.BasicHttpBinding();
+                result.MaxBufferSize = int.MaxValue;
+                result.ReaderQuotas = System.Xml.XmlDictionaryReaderQuotas.Max;
+                result.MaxReceivedMessageSize = int.MaxValue;
+                result.AllowCookies = true;
+                return result;
+            }
+            throw new System.InvalidOperationException(string.Format("Could not find endpoint with name \'{0}\'.", endpointConfiguration));
+        }
+        
+        private static System.ServiceModel.EndpointAddress GetEndpointAddress(EndpointConfiguration endpointConfiguration)
+        {
+            if ((endpointConfiguration == EndpointConfiguration.BasicHttpBinding_ICalculatorService))
+            {
+                return new System.ServiceModel.EndpointAddress("http://localhost/COREWCFSAMPLES/CALCULATORSERVICE/CalculatorService/basicHttp");
```

Comment:
>                return new System.ServiceModel.EndpointAddress("http://localhost/COREWCFSAMPLES/CALCULATORSERVICE/ [](http://example.com/codeflow?start=0&length=114)

Because an upper case path was used, this path is now uppercase.

Reply:
I manually fixed the endpoint address for now.